libraries/AP_RCProtocol/AP_RCProtocol_GHST.cpp

```diff
@@ -177,6 +177,12 @@ void AP_RCProtocol_GHST::_process_byte(uint32_t timestamp_us, uint8_t byte)
         return;
     }
 
+    if (_frame.length < 2) {
```

Comment:
As soon as you say "below" makes me think both here and below should be using a constant, since it seems important that they use the same value.